traveler-swift-core/TravelerKit/Models/CatalogItemDetails.swift

```diff
@@ -20,6 +20,7 @@ public struct CatalogItemDetails: Decodable, Product {
     public let imageUrls: [URL]
     /// Attributes
     public let information: [Attribute]?
+    public var isWishlisted: Bool
```

Comment:
I think an optional value is more suitable here.

When the user is not signed in and model says `false` can be misleading.
unless the API endpoint comes back with false. Maybe we should discuss with backend what is most appropriate

Reply:
Documentation on public properties on functions

Reply:
Currently the API returns 'false' when user is not logged in.

Reply:
This should be optional